ament_cmake_auto/cmake/ament_auto_package.cmake

```diff
@@ -87,6 +90,14 @@ macro(ament_auto_package)
     )
   endif()
 
+  # install folders to share
+  foreach(_folder ${_ARG_INSTALL_TO_SHARE})
+    install(
+      DIRECTORY "${_folder}"
```

Comment:
To avoid the pitfall of passing a directory without a trailing slash this should use a trailing slash.